tortoise/verifying.go

```diff
@@ -0,0 +1,143 @@
+package tortoise
+
+import (
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+func newVerifying(config Config, common *commonState) *verifying {
+	return &verifying{
+		Config:       config,
+		commonState:  common,
+		goodBallots:  map[types.BallotID]struct{}{},
+		layerWeights: map[types.LayerID]weight{},
+		totalWeight:  weightFromUint64(0),
+	}
+}
+
+type verifying struct {
+	Config
+	*commonState
+
+	goodBallots map[types.BallotID]struct{}
+	// weight of good ballots in the layer
+	layerWeights map[types.LayerID]weight
+	// total weight of good ballots from verified + 1 up to last
+	totalWeight weight
+}
+
+func (v *verifying) processLayer(logger log.Log, lid types.LayerID, ballots []tortoiseBallot) {
+	counted, count := v.sumGoodBallots(logger, ballots)
```

Comment:
counted and count are a bit too close. can you rename counted to goodWeight or something?

Reply:
we referred to this weight as counted in all discussions. so i wanted to keep it as is, because there also will be a variable with uncounted weight. i will probably rename count to n or smth